### ISSUE TITLE
Missing accessibility labels

### ISSUE THREAD

**Issue by mhameed** [dkrivoruchko/ScreenStream#150]
Thank you for writing and sharing this application.
The application is generally accessible, but a few buttons are missing accessibility labels.
These labels are read out to blind people when they interact with the application. You can test this out by enabling talkback and interacting with the app.

The buttons that are highlighted in red are those that need labels.
Could you please add them.
Tested with v3.5.3 from fdroid.

Many Thanks
![screen](https://user-images.githubusercontent.com/447017/108512280-e36c3300-72b8-11eb-8962-762da9943522.jpg)

**Comment by dkrivoruchko** [comment]
Yes, will update. Thanks
